AzureMonitorAgent/ama_tst/modules/error_codes.py

```diff
@@ -0,0 +1,24 @@
+# # General Errors
```

Comment:
how are these surfaced to the user of the TST; are they printed in human readable format, are they the TST exit code, etc. ? If they aren't reported in a human-readable way, is the mapping from code -> error description documented anywhere (other than this file) e.g. could it be part of the [.md](https://github.com/Azure/azure-linux-extensions/blob/master/AzureMonitorAgent/ama_tst/AMA-Troubleshooting-Tool.md)